lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -115,6 +134,7 @@ def __init__(self):
         self.machine_id = get_machine_id(node_id)
         self.enclosure_id = None
         self.mode = None
+        self.cvg_count = -1
```

Comment:
Guess we should change this to:
 self.cvg_number = 0

Reply:
No, we are not storing -1 in confstore the -1 value is used to check if user has provided the cvg parameter on command line or not.